demo/values.tpl.yaml

```diff
@@ -54,14 +45,69 @@ dex:
     issuer: http://{{ hostname }}:32002
 
     staticClients:
-    - id: d396912e-2f04-439b-8ae7-d8c585a34790
-      public: true
-      name: 'CLI app'
-      redirectURIs:
-        - 'https://{{ hostname }}:8000/api/auth/device/complete'
-        - https://{{ hostname }}:8000/api/auth/authorize/complete
+      - id: d396912e-2f04-439b-8ae7-d8c585a34790
+        public: true
+        name: "CLI app"
+        redirectURIs:
+          - "https://{{ hostname }}:8000/api/auth/device/complete"
+          - https://{{ hostname }}:8000/api/auth/authorize/complete
```

Comment:
?
```suggestion
          - "https://{{ hostname }}:8000/api/auth/authorize/complete"
```

Reply:
I have no idea, something auto formatted it...